app/experimenter/experiments/tests/test_bugzilla.py

```diff
@@ -111,40 +112,62 @@ def test_create_bugzilla_ticket_creation_failure(self):
         self.assertRaises(BugzillaError, create_experiment_bug, experiment)
 
 
-class TestAddExperimentComment(MockBugzillaMixin, TestCase):
+class TestUpdateExperimentBug(MockBugzillaMixin, TestCase):
 
-    def test_add_bugzilla_comment_pref_experiment(self):
+    def test_update_bugzilla_pref_experiment(self):
         experiment = ExperimentFactory.create_with_status(
             Experiment.STATUS_DRAFT,
             name="An Experiment",
             bugzilla_id="123",
             type=Experiment.TYPE_PREF,
         )
+        cf_tracking = "cf_tracking_firefox{}".format(
+            get_firefox_major_version(experiment.firefox_version)
```

Comment:
Yeah same here, if you hard code the version on the factory you can hard code this whole string.